charters/DataProcessingService/charter.md

```diff
@@ -0,0 +1,65 @@
+# [Data Processing Service](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+
+The Data Processing Service is a service that executes data processing pipelines on DCP data in the Data Storage Service. It can be used in any instance of the Data Coordination Platform (DCP). 
+
+## Objectives
+
+The Data Processing Service aims to develop the infrastructure to run pipelines reliably and at-scale for the DCP. The team working on the service will also operate the infrastructure for the Human Cell Atlas. 
+
+## In-scope
+
+* Creating and maintaining the Data Processing Service  
+
+### Operations
+
+* Operating the instance of the HCA DCP Data Processing Service
+* Initiation and management of the execution of all data processing pipelines that run in the HCA DCP  
+* Establishing and operating automated practices around engineering testing of pipelines in the HCA DCP  
+
+### Documentation
+
+* Creating and outputting metadata on the transformations done to the data in accordance with the HCA DCP Metadata Schema  
+* Documentation on the Data Processing Service and guides on using the service  
+* Creating documentation and scripting around the deployment of the Data Processing Service
+
+## Out-of-scope
+
+* Standing up or operating other instances of the Data Processing Service for other projects  
+
+## Milestones and Deliverables
+
```

Comment:
Is it possible to offer more details (Q1, H2). Otherwise, these read as aspirational for 2019 and 2020.

Reply:
I am struggling to be more specific in this timeframe. I feel even these could change as our current GA and project requirements are rapidly evolving. For example, I have had conversations today about bundle structure which should potentially be added here. I am not sure how to be more specific in 2020 other than general themes; we could be more specific and only speak about 2019 Q1&2 but then the charter would need to be renewed/updated at that point. Happy to talk offline, I may be missing something.